packages/console/src/pages/SignInExperience/components/GuideModal.module.scss

```diff
@@ -42,6 +42,9 @@
   .main {
     display: flex;
     align-items: flex-start;
+    max-width: 1168px;
+    min-width: 604px;
```

Comment:
will it be better to use a mixin for all these max/min-width styles? not a blocker

Reply:
updated